src/modules/Logs/api.ts

```diff
@@ -34,5 +36,7 @@ const request = async (name, path, method, data) => {
     })
 }
 
-// @ts-expect-error
-export const sendLogs = async logs => request('SENDING LOGS', 'log/', 'PUT', logs)
+export const sendLogs = async (logs: LogOutput) => {
+  const payload = JSON.stringify(logs)
```

Comment:
Just checked, and we don't need the extra stringify. The `request` handles that.